h/static/styles/help-page.scss

```diff
@@ -1,10 +1,10 @@
 $base-font-size: 16px;
 $base-line-height: 22px;
 
-@import './reset';
-@import './common';
-@import './elements';
-@import './mixins/icons';
+@import 'reset';
```

Comment:
I know that SASS does not require `./` to specify relative paths but I preferred to use them as it made the syntax we use for relative imports consistent between the JS and SASS code.